src/api/app/models/concerns/configuration_constants.rb

```diff
@@ -42,5 +42,8 @@ module ConfigurationConstants
   ON_OFF_OPTIONS = %i[anonymous default_access_disabled allow_user_to_create_home_project disallow_group_creation
                       change_password hide_private_options gravatar download_on_demand enforce_project_keys
                       cleanup_empty_projects disable_publish_for_branches].freeze
-  PROXY_MODE_ENABLED_VALUES = %i[on ichain mellon].freeze
+
+  module ClassMethods
+    PROXY_MODE_ENABLED_VALUES = %i[on ichain mellon].freeze
```

Comment:
Why do you move this into a sub-module?

Reply:
This `PROXY_MODE_ENABLED_VALUES` constant is only used by a, now, class method: `proxy_auth_mode_enabled?`.

Constants used by instance methods are defined in concerns outside the `module ClassMethods` section of the concern.

Constants used by class methods are defined in concerns inside the `module ClassMethods` section of the concern.

See reference: https://api.rubyonrails.org/classes/ActiveSupport/Concern.html

Reply:
Ah, I see. Thanks!